providers/google/docs/operators/cloud/datacatalog.rst

```diff
@@ -464,112 +322,115 @@ Operators uses a :class:`~google.cloud.datacatalog_v1beta1.types.TagTemplateFiel
 
-.. exampleinclude:: /../../google/tests/system/google/datacatalog/example_datacatalog_tag_templates.py
-    :language: python
-    :dedent: 4
-    :start-after: [START howto_operator_gcp_datacatalog_update_tag_template_field]
-    :end-before: [END howto_operator_gcp_datacatalog_update_tag_template_field]
-
-You can use :ref:`Jinja templating <concepts:jinja-templating>` with
-:template-fields:`airflow.providers.google.cloud.operators.datacatalog.CloudDataCatalogUpdateTagTemplateFieldOperator`
-parameters which allows you to dynamically determine values.
-
 
 .. _howto/operator:CloudDataCatalogDeleteTagTemplateFieldOperator:
 
 Deleting a field
 """"""""""""""""
 
+.. warning::
+    The Data Catalog will be discontinued on January 30, 2026 in favor of Dataplex Universal Catalog. Please use
+    :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogUpdateAspectTypeOperator`.
+
 The :class:`~airflow.providers.google.cloud.operators.datacatalog.CloudDataCatalogDeleteTagTemplateFieldOperator`
 operator delete the tag template field.
 
-.. exampleinclude:: /../../google/tests/system/google/datacatalog/example_datacatalog_tag_templates.py
-    :language: python
-    :dedent: 4
-    :start-after: [START howto_operator_gcp_datacatalog_delete_tag_template_field]
-    :end-before: [END howto_operator_gcp_datacatalog_delete_tag_template_field]
-
-You can use :ref:`Jinja templating <concepts:jinja-templating>` with
-:template-fields:`airflow.providers.google.cloud.operators.datacatalog.CloudDataCatalogDeleteTagTemplateFieldOperator`
-parameters which allows you to dynamically determine values.
-
-
 .. _howto/operator:CloudDataCatalogSearchCatalogOperator:
 
 Search resources
-^^^^^^^^^^^^^^^^
+""""""""""""""""
+
+.. warning::
+    The Data Catalog will be discontinued on January 30, 2026 in favor of Dataplex Universal Catalog. Please use
+    :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogSearchEntriesOperator`.
 
 The :class:`~airflow.providers.google.cloud.operators.datacatalog.CloudDataCatalogSearchCatalogOperator`
 operator searches Data Catalog for multiple resources like entries, tags that match a query.
 
 The ``query`` parameters should defined using `search syntax <https://cloud.google.com/data-catalog/docs/how-to/search-reference>`__.
 
-.. exampleinclude:: /../../google/tests/system/google/datacatalog/example_datacatalog_search_catalog.py
-    :language: python
-    :dedent: 4
-    :start-after: [START howto_operator_gcp_datacatalog_search_catalog]
-    :end-before: [END howto_operator_gcp_datacatalog_search_catalog]
 
-You can use :ref:`Jinja templating <concepts:jinja-templating>` with
-:template-fields:`airflow.providers.google.cloud.operators.datacatalog.CloudDataCatalogSearchCatalogOperator`
-parameters which allows you to dynamically determine values.
-
-The result is saved to :ref:`XCom <concepts:xcom>`, which allows it to be used by other operators.
+Jinja templating usage
```

Comment:
I am not sure that in other examples we have such lists separately for listing operators that use templating. I think we can leave those lines about Xcom on the places where they were before for deprecated operators, and just add some warnings that they were deprecated.